app/grants/clr.py

```diff
@@ -29,9 +29,16 @@
 from grants.models import Contribution, Grant, PhantomFunding
 from perftools.models import JSONStore
 
-LOWER_THRESHOLD = 0.0
 CLR_START_DATE = dt.datetime(2020, 1, 6, 0, 0)
 
+# TODO: MOVE TO DB
+THRESHOLD_TECH = 20.0
```

Comment:
should these be dicts instead of a bunch of varaiables, eg

```
matching_pots = [
{
 'slug': 'tech',
 'amount': 100000,
'config': { 'negative_voting': false },
]
```

Reply:
Yup ! We don't need the -ve voting though -> that is something we'd control via the frontend 
Allow -ve contributions only to tech grants